src/applications/debt-letters/components/DebtLettersList.jsx

```diff
@@ -1,119 +1,129 @@
-                  <tr>
-                    <th className="vads-u-border--0">Date</th>
-                    <th className="vads-u-border--0">Type</th>
-                    <th className="vads-u-border--0">Action</th>
+const DebtLettersList = ({ debtLinks, isVBMSError }) => {
+  const renderAlert = () => (
+    <div
+      className="usa-alert usa-alert-error vads-u-margin-top--0"
+      role="alert"
+    >
+      <div className="usa-alert-body">
+        <h3 className="usa-alert-heading">
+          Your debt letters are currently unavailable
+        </h3>
+        <p className="vads-u-font-family--sans">
+          You can't download your debt letters because something went wrong on
+          our end.
+        </p>
+        <p>
+          <strong>What you can do</strong>
+        </p>
+        <p className="vads-u-font-family--sans">
+          You can check back later or call the Debt Management Center at{' '}
+          <a href="tel: 800-827-0648" aria-label="800. 8 2 7. 0648.">
+            800-827-0648
+          </a>{' '}
+          to find out more information about how to resolve your debt
+        </p>
+      </div>
+    </div>
+  );
+  return (
+    <div className="vads-l-row ">
+      <p className="vads-u-font-size--h2 vads-u-font-weight--bold vads-u-margin-top--1p5 vads-u-margin-bottom--2">
+        Your Debt Letters
+      </p>
+      {isVBMSError && renderAlert()}
+      {!isVBMSError &&
+        debtLinks.length > 0 && (
+          <>
+            <p className="vads-u-margin-y--0 vads-u-font-family--sans">
+              You can view a list of letters sent to your address and download
+              them.
+            </p>
+            <table className="vads-u-font-family--sans vads-u-margin-top--3 vads-u-margin-bottom--0">
+              <thead>
+                <tr>
+                  <th className="vads-u-border--0">Date</th>
+                  <th className="vads-u-border--0">Type</th>
+                  <th className="vads-u-border--0">Action</th>
+                </tr>
+              </thead>
+              <tbody>
+                {debtLinks.map(debtLetter => (
+                  <tr
+                    key={debtLetter.documentId}
+                    className="vads-u-border-top--1px vads-u-border-bottom--1px"
+                  >
+                    <td className="vads-u-border--0">
+                      {moment(debtLetter.receivedAt).format('MMM D, YYYY')}
+                    </td>
+                    <td className="vads-u-border--0">
+                      {debtLetter.typeDescription}
+                    </td>
+                    <td className="vads-u-border--0">
+                      <a
+                        href={encodeURI(
+                          `${environment.API_URL}/v0/debt_letters/${
+                            debtLetter.documentId
+                          }`,
+                        )}
+                      >
+                        Download Letter
+                      </a>
```

Comment:
I'm not sure if this will work here, but there are some accessibility recommendations for download links - https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/152#issuecomment-553127340